libraries/core/Cargo.toml

```diff
@@ -7,6 +7,6 @@ license = "Apache-2.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
We should probably rename the package from `dora-common` to `dora-core` too?

Reply:
Yep done.